src/schema/keywords.ts

```diff
@@ -46,6 +46,11 @@ export const typeDefs = /* GraphQL */ `
     Description of the keyword
     """
     description: String
+
+    """
+    Roadmap.sh link for the keyword
+    """
+    roadmap: String
```

Comment:
Its a string but maybe we want to link the slug and then add domain through field resolver just so we don't hardcode it inside the db?